index.d.ts

```diff
@@ -1446,22 +1446,28 @@ declare namespace Eris {
 
   export class Invite implements SimpleJSON {
     code: string;
-    channel: { id: string; name: string };
+    channel: {
+      id: string;
+      name: string;
+      type: 0 | 2 | 4 | 5 | 6;
+    };
     guild: {
```

Comment:
Im not sure if we are still supporting this but according to the discord docs you can also have Invites for Group DMs. This means guild could be optional.

![image](https://user-images.githubusercontent.com/23035000/71998151-1cf4f280-320d-11ea-9f0d-300b83d44ef6.png)

If this is accurate and it is a "partial", if memory serves the only guaranteed props on partials are ids. This means all props like name and features should be optional 🤔 Not sure on this.

Reply:
guild has been made optional